ui/js/components/perf/compare.js

```diff
@@ -69,6 +69,75 @@ treeherder.component('phCompareTable', {
     }
 });
 
+treeherder.component('phTrendTable', {
```

Comment:
(I'm not 100% sure that makes sense, but I think it does... yes, you would use ng-if here for conditional behaviour)